src/domain/callout/creation-dialog/useCalloutCreation/useCalloutCreation.ts

```diff
@@ -79,7 +79,7 @@ export const useCalloutCreation = (initialOpened = false): CalloutCreationUtils
         fields: {
           callouts(existing = []) {
             const newCalloutRef = cache.writeFragment({
-              data: createCalloutOnCollaboration,
+              data: { aspects: [], canvases: [], ...createCalloutOnCollaboration },
```

Comment:
You can select the aspects and canvases in the query and don't bother anymore.
So create a fragment and use it both in the query and the mutation if they don't diverge that much.